components/core/email.conf

```diff
@@ -0,0 +1 @@
+{"username": "bassa.ucsc@gmail.com", "password": "bassa.123"}
```

Comment:
You mean to remove config files from Git? Yes, agree on that. However, place an example config file (i.e. email.conf.example) so that people can change it on their instances.